modules/guides/pages/updating-data.adoc

```diff
@@ -26,62 +25,46 @@ include::partial$data-warning.adoc[]
 
 To update a document, or create the document if it doesn't exist, perform an upsert operation.
 
-[{tabs}]
+[tabs]
 ====
-ifdef::flag-devex-command-line[]
-cbc::
+cbsh::
 +
 --
-. Create a JSON document containing the updated data.
+. If necessary, use `cb-env` to set the bucket, scope, and collection in which to store the document.
```

Comment:
This may be worth changing. A bucket will always need to be set when performing any `doc`command. After that setting the scope/collection is optional, since if they aren't set then they are set to _default, this can be seen as follows:

```
👤 Westwooo 🏠 local
> cb-env bucket travel-sample
👤 Westwooo 🏠 local in 🗄 travel-sample._default._default
```

Reply:
Done